### PR TITLE
Note incompatibility with Marshmallow and Lollipop

### PR DESCRIPTION
**Pull request by coreyweiner**

### CODE REVIEW

File: UsageGuide.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # HyprMX Android Usage Guide
 
-Now that you have the HyprMX library installed in your project, it's time to put it to work! To help you integrate HyprMX into your application, we will first provide a brief overview of the workflow and concepts, and then concrete examples of how to use the library.
+Now that you have the HyprMX library installed in your project, it's time to put it to work! To help you integrate HyprMX into your application, we will first provide a brief overview of the workflow and concepts, and then concrete examples of how to use the library. Please note that the current release of our Android SDK does not serve ads on Lollipop or Marshmallow due to a bug in Google's YouTube API which can cause crashes on these OS versions.
```
Comment (jzisserJg) [inline UsageGuide.md]: @coreyweiner , on second thought after reading this, I think we should also note that the experience for the user on a Lollipop or Marshmallow device will be as if there are no available offers for them.

Comment (coreyweiner) [inline UsageGuide.md]: Yes I was thinking the same, I modified your original language to address this concern. Did I fail?

Comment (jzisserJg) [inline UsageGuide.md]: That's excellent work, @coreyweiner !

**Comment by daveb-jungroup** [comment]
looks good to me.

**Pull request merged**